c_api/src/lib.rs

```diff
@@ -0,0 +1,29 @@
+// Copyright 2023 Adobe. All rights reserved.
+// This file is licensed to you under the Apache License,
+// Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+// or the MIT license (http://opensource.org/licenses/MIT),
+// at your option.
+// Unless required by applicable law or agreed to in writing,
+// this software is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR REPRESENTATIONS OF ANY KIND, either express or
+// implied. See the LICENSE-MIT and LICENSE-APACHE files for the
+// specific language governing permissions and limitations under
+// each license.
+
+/// This module exports a C2PA library
+mod c2pa_stream;
+mod c_api;
+mod error;
+#[cfg(feature = "json_api")]
```

Comment:
I like the JSON API. Many other languages now natively or quite well use JSON. Should we enable it by default? (Especially since we offer to create SignerInfo from the json. It increases API surface, but it also makes it feel more... complete?)

Reply:
This json_api feature is for a set of Rust only functions I provided to mirror the exposed C API. I did it so that the python implementation could share some code. Now it serves no purpose and it only includes the older v1 style api functions.